.github/workflows/test.yml

```diff
@@ -17,29 +17,18 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - '20.9'
-          - '18.17'
-          - '16.20'
-          - '14.16'
+          - '22.13'
```

Comment:
Something I've thought about in the past (and I feel like @BlackHole1 has pointed this out before), is we should probably explicitly test our lower bound Node.js version to ensure we remain compatible with it. It's extra CI runs, but since we just reduced the matrix here we'll still have a net reduction if we add back '20.0.0' to the matrix.

Reply:
Do you mean the lower bound being `22.0.0` (minimum engines value)?

Reply:
Yes, typo on my part. 🙁